crm_phonecall/models/crm_phonecall.py

```diff
@@ -125,10 +127,10 @@ def write(self, values):
 
     @api.multi
     def compute_duration(self):
+        """Calculate duration based on phonecall date."""
         for phonecall in self.filtered('date'):
-            if phonecall.duration <= 0:
-                duration = datetime.now() - datetime.strptime(
-                    phonecall.date, DEFAULT_SERVER_DATETIME_FORMAT)
+            if phonecall.duration <= 0 and phonecall.date:
+                duration = datetime.now() - phonecall.date
```

Comment:
Better to use `fields.Datetime.now()` for taking advantage of new stuff to come for "mocking" times if needed.

Reply:
Right @pedrobaeza